coffee_roaster_market/apps/account/views.py

```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth import get_user_model
+from rest_framework import viewsets, status
+from rest_framework.decorators import action
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework import permissions
+
+from .serializers import UserActivationSerializer, UserSerializer, UserCreateSerializer
+from .permissions import CurrentUserOrAdmin
+
+user = get_user_model()
+
+
+class ApiUsersView(viewsets.ModelViewSet):
+    queryset = user.objects.all()
+
+    def get_serializer_class(self):
```

Comment:
Same as https://github.com/AleksanderPawlak/coffee-roaster-market/pull/12#discussion_r654963786

Reply:
Add in entire file.